README.md

```diff
@@ -61,82 +63,51 @@ You need to specify your billing key **only** if you need Android support.
+```
+	{
+		platform: "ios" or "android",
+		orderId: transaction identifier for iOS or order id for Android,
+		receipt: purchaseToken or ios receipt as String,
+		productId: "sword001",
+		packageName: "jp.wizcorp.game"
+	}
+```
 
-This method for Android fetches any pending consumption products.
+## JavaScript APIs
 
-- *Return* success with an Array of one or more Objects containing product information that can be used when consuming or verification.
-	e.g.
+### getPendingPurchases(Function success, Function failure)
 
-		[
-			{
-				platform: "ios" or "android",
-				orderId: transaction identifier for iOS or order id for Android,
-				receipt: purchaseToken or ios receipt as String,
-				productId: "sword001",
-				packageName: "jp.wizcorp.game"
-			},
-			{
-				...
-			} ...
-		]
+Get a list of purchases which have not been ended yet using ```finishPurchase```.
 
+- *Return* success with an Array of one or more ```Purchase``` objects.
 - *Return* failure with error
 
-### restoreAll(Function success, Function failure)
-
-Get a list of non-consumable item receipts / purchaseTokens
+Developers should check any returned items with server APIs and complete their purchase using ```finishPurchase```.
 
-iOS should internally...
-Check the call the code below return any owned non consumables etc.
+### restoreAllPurchases(Function success, Function failure)
 
-`[[SKPaymentQueue defaultQueue] addTransactionObserver:self];`
-`[[SKPaymentQueue defaultQueue] restoreCompletedTransactions];`		
-then `(void)paymentQueueRestoreCompletedTransactionsFinished`
-
-Android should internally call `Bundle getPurchases()` on IInAppBillingService.aidl Class
-This populates an Inventory object with all purchases ever made except the consumed purchases.
-
-- *Return* success with an Array of one or many puchaseTokens (Android) or receipts (iOS)
-	* e.g. Android
-		`[ "puchase-token-string", ... ]`
+Get a list of previous purchases of non-consumable and not yet finished purchases.
 
+- *Return* success with an Array of one or more ```Purchase``` objects.
 - *Return* failure with error
 
-(Developer should check any returned items with server APIs. If any items exist that are consumables, but have not been comsumed. The developer should call `consumePurchase()` because it is likely a previous purchase was not completed )
-
+Developers should check any returned items with server APIs. If any items that exist are consumables but have not been consumed, the developer should consume them using ```finishPurchase``` because it is likely that a previous purchase was not completed.
 
 ### makePurchase(String productId, Function success, Function failure)
 
-Make a purchase given a product Id (Quantity is not settable with the API, it is always 1 to be cross platform complete).
+Make a purchase given a product id (Quantity is not settable with the API, it is always 1 to be cross-platform complete).
```

Comment:
I was also not a fan of writing `Id` in docs. But according to the dictionary, `ID` is the right way to write it.